acceptance/bundle/resource_deps/bad_syntax/out.plan.terraform.txt

```diff
@@ -8,3 +8,4 @@ Error: Invalid attribute name
 An attribute name is required after a dot.
 
 
+No changes. Your infrastructure matches the configuration.
```

Comment:
Would it be more informational to show count there as well? and perhaps normalize the message to 

with changes:

> Plan: 4 to add, 0 to change, 0 to delete, 26 unchanged

without changes:

> Plan: 0 to add, 0 to change, 0 to delete, 30 unchanged

Reply:
Yeah I like having it at the last line. The problem is just this extra `Building python_artifact...` like that makes the output slightly messy :-/ That's why I'd propose to do the summary line at the top, signaling a start of the (more) useful output of the command.